### PR TITLE
home carousel

### PR DESCRIPTION
**Pull request by dev-rajneeshkumar**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--moleculardevices--hlxsites.hlx.page/
- After: https://home-changes--moleculardevices--hlxsites.hlx.page/

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -2103,19 +2103,9 @@ main .columns-separator-line .columns-wrapper .columns {
...
-.image-position-right img {
-  object-position: right;
-}
-
-.image-position-left img {
-  object-position: left;
-}
-
-.image-position-center img {
-  object-position:  center;
```
Comment (mhaack) [inline styles/styles.css:2120]: How is this related to the homepage change? Especially `.application-note main .picture img` is not related to the homepage.

Comment (dev-rajneeshkumar) [reply styles/styles.css:2120]: This piece of code is already live for appnot images fix. https://main--moleculardevices--hlxsites.hlx.page/en/assets/app-note/dd/img/3d-quantitative-measurement-of-t-cell-penetration-into-cancer-spheroids so I think we can go ahead

Comment (mhaack) [reply styles/styles.css:2120]: Ok.

**Review by mhaack — APPROVED** [review]
